### PR TITLE
Hide rsvp buttons in past events

### LINKED ISSUE #1703: Hide RSVP buttons if the event is in the past
## Description of the issue 📄

Currently you can RSVP to past "Events"

## Screenshots 📷
screencapture-codebar-io-events-speedrunning-web-development-with-anvil-2022-01-19-14_18_36

## Steps to fix 🛠

Update some of the logic in this file `app/views/events/_event_actions.html.haml`

## To do 📋

* [ ] Claim this issue (comment below, or assign yourself if you are part of the codebar org)
* [ ] Fork and clone the repository
* [ ] Update the relevant files. Follow the steps to fix section in this issue.
* [ ] Commit your changes as one commit. Use the title of this issue as your commit message
* [ ] Submit a pull request
* [ ] Mention this issue in the PR description by including it's number
* [ ] Have your pull request reviewed & merged by a codebar team member

### PR DESCRIPTION
**Pull request by joaoguiIherme**
Closes #1703 

This PR resolves the issue 1703 which mentions users can RSVP to past "Events".

### CODE REVIEW

**Comment by matyikriszta** [comment]
@joaoguiIherme could you add an integration test to this PR please?

**Comment by joaoguiIherme** [comment]
> @joaoguiIherme could you add an integration test to this PR please?

Done!

**Comment by joaoguiIherme** [comment]
@gnclmorais what can i do about this issue (numbers of methods in `Event`class) reported by codeclimate? What's the best approach in your opinion?

**Comment by gnclmorais** [comment]
I’m not sure I’m going to be useful here, since I think 20 is a very low number of the number of methods… I think we’re a bit too strict here. 😅

**Comment by matyikriszta** [comment]
@joaoguiIherme @gnclmorais alternatively yes, we maybe should just ignore the warning, 20 is really low. Certainly some other models have way more classes. I can mark this issue as won't fix for now and see about updating the config.

**Comment by joaoguiIherme** [comment]
imho, these [methods](https://github.com/codebar/planner/pull/1709/files#diff-716ef9c1cac9e66383f5c4d8a13df148b3b8487421f4bce8e82a367507d5c2edR114-R127) could be moved to a concern, but i agree that there aren't that many methods.

**Review by matyikriszta — APPROVED** [review]
@joaoguiIherme it took me a while to get around to it but I marked the Codeclimate issue as Won't fix for now. This is good to go I think. Thank you for your contribution.

@gnclmorais could you give this PR just one more once over and let me know if you approve as well? Would like to have a second review on this one.

**Review by gnclmorais — APPROVED** [review]
Code looks nice & tight, and the tests are relevant and pass — I’m really happy with this! _Obrigado, @joaoguiIherme!_

**Pull request merged**
